### PR TITLE
python: update deprecated script/file path

### PR DESCRIPTION
**Pull request by antoine-amara**
After a discussion with @navarroaxel , I fix the python deprecated documentation,

* update deprecated script path for each python command example.
* replace {{script.py}} to {{path/to/file.py}}

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@antoine-amara this PR has a Git conflict. Could you please fix it?

**Comment by mfrw** [comment]
@antoine-amara Please feel free to let us know if you need help in resolving a merge conflict, we will be more than happy to give pointers on what to do :)

[Just in case] :)

**Comment by antoine-amara** [comment]
I update my branch and I resolve the conflict @navarroaxel , @mfrw .

File: pages/common/python.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
 - Interactively debug a Python script:
 
-`python -m pdb {{script.py}}`
+`python -m pdb {{path/to/file.py}}`
```
Comment (marchersimon) [inline pages/common/python.md:32]:
```suggestion
`python -m {{pdb}} {{path/to/file.py}}`
```
Could you please also put `pip` in line 28 in `{{}}`? I can't make a suggestion there.

**Review by marchersimon — COMMENTED** [review]
I'm not sure about the `script.py` → `file.py` change. What do you think about using `path/to/script.py`?

**Comment by navarroaxel** [comment]
> I'm not sure about the `script.py` → `file.py` change. What do you think about using `path/to/script.py`?

We use `file` in the `node` page.

**Comment by marchersimon** [comment]
Is there a reason for this? `script` would be more descriptive.

**Comment by navarroaxel** [comment]
every `py` or `js` files are script.

**Comment by marchersimon** [comment]
But `script.py` or `script.js` would make it even more clear. Especially for beginners.

**Comment by antoine-amara** [comment]
thanks for the review @marchersimon . In my opinion, we should keep `{{file.py}}` because this pattern is used in multiple command descriptions. (see autoflake for a python command example).

**Review by marchersimon — APPROVED** [review]
Thanks @antoine-amara

**Pull request merged**